.github/workflows/create_release.yml

```diff
@@ -0,0 +1,77 @@
+name: Create Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "New release version (e.g. 0.8.0 or 0.8.1)"
+        required: true
+
+jobs:
+  set_version:
+    runs-on: [self-hosted-org, linux]
+    steps:
+    - name: Checkout this repo
+      uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.NOVUM_PRIVATE_REPOS }}
+
+    - name: Update medatata.json version
+      uses: jossef/action-set-json-field@v1
+      with:
+        file: metadata.json
+        field: version
+        value: ${{ github.event.inputs.version }}
+
+    - id: show_diff
+      name: Show git diff
+      run: |
+        git diff
+
+    - name: Commit & Push changes
+      id: push_version
+      uses: EndBug/add-and-commit@v7
+      with:
+        github_token: ${{ secrets.NOVUM_PRIVATE_REPOS }}
+        branch: 'master'
+        message: 'Bump version ${{ github.event.inputs.version }}'
+        author_email: lifecycle-github@telefonica.com
+        author_name: tuentisre
```

Comment:
Should this be considered as a secret to avoid spam?
as this is a public repo...